workflow-bot-app/src/test/java/com/symphony/bdk/workflow/IntegrationTest.java

```diff
@@ -118,6 +123,10 @@ protected static Stream stream(String streamId) {
   @BeforeEach
```

Comment:
After first commit, tests were broken because the injected bdk services were null. 
I came up with this solution to fix the tests, but I'm not convinced it is the best way. see my commit message here https://github.com/finos/symphony-wdk/pull/24/commits/715252cfbe2a3124a812432ab96cf8db5bcfeabe